### DISCUSSION TITLE
Release availability

### DISCUSSION THREAD

**Discussion by trehman-gsma** [camaraproject/DeviceIdentifier#56] [General]
Hi @Kevsy @eric-murray 

Thanks for the great work in maintaining this repository. Do you have any thoughts on how close DeviceIdentifier is to having a release?

We have a couple of operators who are interested in implementing DeviceIdentifier - but it doesn't seem advisable to implement the main branch as it is subject to change.

cc @deeokonkwo

**Comment by eric-murray** [comment]
Hi @trehman-gsma 

I believe PR #55 is the last required before a release. There is a discussion as to whether the YAML should be separated into separate YAMLs for the two defined endpoints, but initially I am happy to release a single YAML.

PR #55 should be approved at the next meeting (Friday 8th), with the release shortly thereafter.

**Comment by trehman-gsma** [comment]
Thanks @eric-murray!

I will close this discussion - no further questions or comments from me.

cc @deeokonkwo
